library/src/main/java/com/novoda/downloadmanager/LiteDownloadFileId.java

```diff
@@ -2,41 +2,40 @@
 
 final class LiteDownloadFileId implements DownloadFileId {
 
-    private final int id;
+    private final String id;
 
-    LiteDownloadFileId(int id) {
+    LiteDownloadFileId(String id) {
         this.id = id;
     }
 
+    @Override
+    public String rawId() {
+        return String.valueOf(id);
```

Comment:
This is not needed anymore... now you can just return `id`

Reply:
c3e7df3